src/service.rs

```diff
@@ -1187,6 +1189,7 @@ impl<'a> From<&'a Services::SERVICE_STATUS_PROCESS> for ServiceExitCode {
 
 bitflags::bitflags! {
     /// Flags describing accepted types of service control events.
+    #[derive(Debug, Clone, PartialEq, Eq, Hash)]
```

Comment:
Could you derive `PartialOrd`/`Ord` here to prevent breaking changes?

```rust
#[derive(PartialEq, Eq, PartialOrd, Ord, Hash, Debug, Clone, Copy)]
```

Reply:
Done.